### DISCUSSION TITLE
How do i install ha-Fusion?

### DISCUSSION THREAD

**Discussion by arn35** [matt8707/ha-fusion#408] [Q&A]
Hello,

I have issue installing Ha-fussion or more correct i have installed it but i have an issue. Im getting "Invalid redirect URI" when trying to open it up, what have i missed and how do i proceed? 
Im new to HA and im running it on an RPI3.

**Comment by arn35** [comment]
This is how it looks for me..
![Screenshot_20240306_184553_Home Assistant](https://github.com/matt8707/ha-fusion/assets/69081779/54599bbd-1b9f-499f-96c8-a1156d5486de)

**Comment by XtosX** [comment]
I have also encountered the same issue, not sure what the reason is. Accessing Fusion on PC is normal, but there is an error message on the mobile phone.
